pkg/logging/log.go

```diff
@@ -22,6 +22,8 @@ const (
 	TextFormat = "text"
 )
 
+var globalLog = log.Log
```

Comment:
Can you add a comment explaining why this is needed ?

Reply:
yes, i'm on it

Reply:
👍